tico/experimental/quantization/passes/remove_weight_dequant_op.py

```diff
@@ -101,23 +127,32 @@ def call(self, exported_program: ExportedProgram) -> PassResult:
             if QPARAM_KEY in q_weight.meta:
                 continue
 
-            q_weight_val = q_weight.meta["val"]
-            assert isinstance(q_weight_val, FakeTensor)
+            q_weight_meta = q_weight.meta["val"]
+            assert isinstance(q_weight_meta, FakeTensor)
```

Comment:
I renamed `q_weight_val` to `q_weight_meta`, because I use `q_weight_val` as a variable name for the actual value of weights.